doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='setup-auto-cleanup-proc']
+= Setting up automatic cleanup job
+
+You can set up an automatic cleanup job in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. Click *New Job*.
+. Enter a name, due date and time and following into the *Type* text field.
```

Comment:
Just a question:

In this Enter a name, due date and time and following into the *Type* text field, what is the following? Is it a command that we are including in the *Type* text field?

**Suggestion:** Enter a name, due date, time and, the following command into the *Type* text field.

Reply:
yes, following is the command.
Have made the suggested change